nsxt/resource_nsxt_policy_parent_security_policy_test.go

```diff
@@ -0,0 +1,167 @@
+	name := getAccTestResourceName()
+	testResourceName := "nsxt_policy_parent_security_policy.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		CheckDestroy: func(state *terraform.State) error {
+			return testAccNsxtPolicyParentSecurityPolicyCheckDestroy(state, name)
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNsxtPolicyParentSecurityPolicyTemplate(false, name, "true", "1", "true"),
+			},
+			{
+				ResourceName:      testResourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccResourceNsxtPolicyImportIDRetriever(testResourceName),
+			},
+		},
+	})
+}
+
+func TestAccResourceNsxtPolicyParentSecurityPolicy_importBasic_multitenancy(t *testing.T) {
+	name := getAccTestResourceName()
+	testResourceName := "nsxt_policy_parent_security_policy.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccOnlyMultitenancy(t)
+		},
+		Providers: testAccProviders,
+		CheckDestroy: func(state *terraform.State) error {
+			return testAccNsxtPolicySecurityPolicyCheckDestroy(state, name, defaultDomain)
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNsxtPolicyParentSecurityPolicyTemplate(true, name, "true", "1", "true"),
+			},
+			{
+				ResourceName:      testResourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccResourceNsxtPolicyImportIDRetriever(testResourceName),
+			},
+		},
+	})
+}
+
+func testAccNsxtPolicyParentSecurityPolicyCheckDestroy(state *terraform.State, displayName string) error {
+	connector := getPolicyConnector(testAccProvider.Meta().(nsxtClients))
+	for _, rs := range state.RootModule().Resources {
+
+		if rs.Type != "nsxt_policy_parent_security_policy" {
+			continue
+		}
+
+		resourceID := rs.Primary.Attributes["id"]
+		domain := rs.Primary.Attributes["domain"]
+		exists, err := resourceNsxtPolicySecurityPolicyExistsInDomain(testAccGetSessionContext(), resourceID, domain, connector)
+		if err != nil {
+			return err
+		}
+		if exists {
+			return fmt.Errorf("Policy SecurityPolicy %s still exists", displayName)
+		}
+	}
+	return nil
+}
+
+func testAccNsxtPolicyParentSecurityPolicyTemplate(withContext bool, name, locked, seqNum, tcpStrict string) string {
+	context := ""
+	if withContext {
+		context = testAccNsxtPolicyMultitenancyContext()
+	}
+	return testAccNsxtPolicySecurityPolicyDeps() + fmt.Sprintf(`
+resource "nsxt_policy_parent_security_policy" "test" {
+%s
+  display_name    = "%s"
+  description     = "Acceptance Test"
+  domain = "default"
+  category        = "Application"
+  locked          = %s
+  sequence_number = %s
+  stateful        = "true"
+  tcp_strict      = %s
+  scope           = [nsxt_policy_group.group1.path]
+
+  tag {
+    scope = "color"
+    tag   = "orange"
+  }
+
+  depends_on = [nsxt_policy_group.group1]
```

Comment:
there is an implicit dependency for the group, so explicit one shouldn't be required?

Reply:
Done.